cardano-cli/src/Cardano/CLI/EraBased/Transaction/Run.hs

```diff
@@ -1692,7 +1717,8 @@ runTransactionCalculatePlutusScriptCostCmd
       tx
    where
     calculatePlutusScriptsCosts
-      :: CardanoEra era
+      :: forall era
+       . CardanoEra era
```

Comment:
Something is wrong if `CardanoEra era` exists here. Plutus scripts become available in Alonzo.

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/1079/commits/288572321a7dccb4f0122cd722449f363ec7b436